lib/eslint/eslint-helpers.js

```diff
@@ -630,17 +630,20 @@ function isErrorMessage(message) {
  * Returns result with warning by ignore settings
  * @param {string} filePath File path of checked code
  * @param {string} baseDir Absolute path of base directory
+ * @param {boolean} configured A boolean value indicating if a configuration for the file was found.
  * @returns {LintResult} Result with single warning
  * @private
  */
-function createIgnoreResult(filePath, baseDir) {
+function createIgnoreResult(filePath, baseDir, configured = true) {
     let message;
     const isInNodeModules = baseDir && path.dirname(path.relative(baseDir, filePath)).split(path.sep).includes("node_modules");
 
     if (isInNodeModules) {
         message = "File ignored by default because it is located under the node_modules directory. Use ignore pattern \"!**/node_modules/\" to disable file ignore settings or use \"--no-warn-ignored\" to suppress this warning.";
-    } else {
+    } else if (configured) {
         message = "File ignored because of a matching ignore pattern. Use \"--no-ignore\" to disable file ignore settings or use \"--no-warn-ignored\" to suppress this warning.";
+    } else {
+        message = "File ignored because no matching configration was supplied.";
```

Comment:
```suggestion
        message = "File ignored because no matching configuration was supplied.";
```

Typo.

Reply:
Fixed in d15c087ca40908678c8eb02ccfecde9e21e0b17b, thanks.